code/src/components/ToDos.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { useDispatch } from 'react-redux'
+import styled from 'styled-components/macro'
+import moment from 'moment'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faTrashAlt } from '@fortawesome/free-solid-svg-icons'
```

Comment:
I did not think that there is a package on npm for font awesome. I put a link in public index.html and then used `<i> `tags for icons. But I think this approach is more relevant for react, because react has a lot of packages and it is better to be consistent and use the same for everything, like in this case package for font awesome.

Reply:
I found this approach on the [Fontawesome-pages](https://fontawesome.com/v5.15/how-to-use/on-the-web/using-with/react) 🙂